modules/apps/foundation/password-policies-admin/password-policies-admin-web/bnd.bnd

```diff
@@ -1,6 +1,8 @@
 Bundle-Name: Liferay Password Policies Admin Web
 Bundle-SymbolicName: com.liferay.password.policies.admin.web
 Bundle-Version: 1.0.0
+Include-Resource: @com.liferay.portal.impl-*.jar!/com/liferay/portal/model/PasswordPolicyImpl.class,\
+
```

Comment:
@JorgeFerrer @rotty3000 I found a better way than this.  I'll resend to you, Jorge.